torch/csrc/jit/codegen/cuda/test/test_gpu_gather_ops.cpp

```diff
@@ -0,0 +1,335 @@
+#if defined(USE_CUDA)
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <torch/csrc/jit/codegen/cuda/kernel_cache.h>
+#include <torch/csrc/jit/codegen/cuda/arith.h>
+#include <torch/csrc/jit/codegen/cuda/executor.h>
+#include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
+#include <torch/csrc/jit/codegen/cuda/ir_builder.h>
+#include <torch/csrc/jit/codegen/cuda/scheduler/all_schedulers.h>
+
+#include <test/cpp/jit/test_utils.h>
+#include <torch/csrc/jit/codegen/cuda/test/test_gpu_validator.h>
+#include <torch/csrc/jit/codegen/cuda/test/test_utils.h>
+
+#include <torch/torch.h>
+
+namespace torch {
+namespace jit {
+
+using namespace torch::jit::fuser::cuda;
+
+// pass
+TEST_F(NVFuserTest, TorchGatherOpAllDim_CUDA) {
```

Comment:
Our test naming convention is `Fusion***_CUDA`. Please add `Fusion` to the test names